platform-configs/novacustom-v560tnd.robot

```diff
@@ -16,11 +16,3 @@ ${CPU_MAX_FREQUENCY}=               4800
 ${CPU_MIN_FREQUENCY}=               200
 
 ${NVIDIA_GRAPHICS_CARD_SUPPORT}=    ${TRUE}
-
-${POWER_CTRL}=                      none
```

Comment:
@JanPrusinowski I wonder why do we need such a change in this MR?

Reply:
No we dont. I have cleaned the Commits - rebased to current develop. I noticed that some of my fixes from this PR were merged already with other issues.